File: observer/Samuel Sales/DesafioObserver_Samuel/src/Canais/Viva.java
package Canais;

import agencia.Agencia;

/**
 *
 * @author Samuel Sales - Facul
 */
public class Viva implements ObserverGrupoGlobo {

    @Override
    public void atualizar(Agencia entrada) {
        System.out.println(entrada.getNome() + ": Canal Viva");
    }

}

File: observer/PadraoObserverII/PadraoObserverII/Canais/Viva.cs
using PadraoObserverII.Interface;
using System;

namespace PadraoObserverII.Canais
{
    class Viva : IObserverCanal
    {
        public void Atualizar(Agencia entrada)
        {
            Console.WriteLine(entrada.Nome + ": Canal VIVA!");
        }
    }
}
